src/custom/components/Menu/MenuMod.tsx

```diff
@@ -144,9 +145,10 @@ export default function Menu(props?: PropsWithChildren<void>) {
             <ButtonPrimary onClick={openClaimModal} padding="8px 16px" width="100%" borderRadius="12px" mt="0.5rem">
               Claim UNI
             </ButtonPrimary>
-          )} */}
+          )}
         </MenuFlyout>
-      )}
+        */
+        props?.children}
```

Comment:
nit: imo easier to read if `props?.children` is above the commented `<MenuFlyout/>` tho i sort of see why u wrote it like this